daily_post.py

```diff
@@ -45,7 +46,13 @@ def gen_post():
     j2_env = Environment(loader=FileSystemLoader('./templates'),trim_blocks=True)
     template = j2_env.get_template('daily_post.j2')
 
-    return template.render(forecast=get_weather())
+    quote_of_the_day = ""
+    if now.weekday() == 4:  # Friday is Fri-ku-day
+        quote_of_the_day = daily_markov.get_haiku(subreddit)
+    else:
+        quote_of_the_day = daily_markov.get_quote(subreddit, SUB)
+
+    return template.render(forecast=get_weather(), qotd=quote_of_the_day)
```

Comment:
Do we want to send in the weekday as well, so we can put a different thing in the render for haiku?

If so, let's be sure to put the weekday in a  CONST variable at the top of the script since it would be used in multiple places. To pass it into the renderer just add it as a new param

```
return template.render(day_of_week=WEEKDAY, forecast=get_weather(), qotd=quote_of_the_day)
```